packages/cmd/src/doc/option/ICommandOptionDefinition.ts

```diff
@@ -18,9 +18,10 @@ import { ICommandOptionValueImplications } from "./ICommandOptionValueImplicatio
  * "existingLocalFile": a file for which fs.existsSync returns true
  * "json": a parseable JSON string
  * "array": an array of strings
+ * "stringOrEmpty" : allow string to be empty when defined
  */
 export type CommandOptionType
-    = "string" | "number" | "json" | "existingLocalFile" | "boolean" | "count" |
+    = "string" | "number" | "json" | "existingLocalFile" | "boolean" | "count" | "stringOrEmpty" |
```

Comment:
I would define `stringOrEmpty` at the end of the list after array since that is how it is ordered in the doc.